### ISSUE TITLE
bug with ReferencedList

### ISSUE THREAD

**Issue by cvlmtg** [marmelab/ng-admin#229]
I've had some issues with ReferencedList not being able to generate the correct REST request when showing a record. moreover when it happens ng-admin just hangs without showing any error message (and it seems to happen in other places when the server returns something different from a 200 status code)

this little patch seems to have solved the problem, but I'm not sure it is the correct or intended way to fix it.

```patch

diff --git a/src/javascripts/ng-admin/Crud/repository/RetrieveQueries.js b/src/javascripts/ng-admin/Crud/repository/RetrieveQueries.js
index dba756e..1baa2ec 100644
--- a/src/javascripts/ng-admin/Crud/repository/RetrieveQueries.js
+++ b/src/javascripts/ng-admin/Crud/repository/RetrieveQueries.js
@@ -101,7 +101,7 @@ define(function (require) {
                 filterName;
             params._filters = {};
             for (filterName in filters) {
-                if (filterFields[filterName].hasMaps()) {
+                if (filterFields[filterName] && filterFields[filterName].hasMaps()) {
                     angular.extend(params._filters, filterFields[filterName].getMappedValue(filters[filterName]));
                 } else {
                     params._filters[filterName] = filters[filterName];
@@ -209,7 +209,7 @@ define(function (require) {
             filter = {};
             filter[referenceList.targetReferenceField()] = entityId;

-            calls.push(self.getRawValues(referenceList.getReferencedView(), 1, null, sortField, sortDir, filter));
+            calls.push(self.getRawValues(referenceList.getReferencedView(), 1, filter, sortField, sortDir));
         }

         return this.$q.all(calls)
````

**Comment by manuquentin** [comment]
Thanks for your report !

It will be fixed in #233

**Comment by cvlmtg** [comment]
thanks for your support :)
